dear_petition/petition/types/tests/test_not_guilty.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from dear_petition.petition import constants
+from dear_petition.petition.tests.factories import (
+    CIPRSRecordFactory,
+    OffenseFactory,
+    OffenseRecordFactory,
+    PetitionFactory,
+)
+
+pytestmark = pytest.mark.django_db
+
+
+def test_charged_not_guilty_record(batch, record1):
+    """Charged not guilty records should be included"""
+    offense = OffenseFactory(verdict="Not Guilty", ciprs_record=record1)
```

Comment:
Is there a difference between this `offense` and the test fixture `not_guilty_offense`?

Reply:
No, good call. I'll use that.